components/Registration/index.tsx

```diff
@@ -350,13 +354,19 @@ const Registration: FunctionComponent<RegistrationProps> = ({
     try {
       // Retreiving Claim Signature from the backend signer service
       const claimDetails = await getClaimDetails();
-
```

Comment:
Remove console.log

Reply:
will rmeove in next PR , need this for debugging just in case